tests/checkpoint/test_checkpoint_config.py

```diff
@@ -1,59 +0,0 @@
-import pytest
-
-from great_expectations.checkpoint import Checkpoint
-
-DATA_CONTEXT_ID = "00000000-0000-0000-0000-000000000001"
-
-
-@pytest.fixture
```

Comment:
Checkpoint config doesn't exist (Pydantic model joins the config and runtime obj)